tests/experimental/datasources/test_pandas_filesystem_datasource.py

```diff
@@ -158,7 +161,7 @@ def test_data_asset_defined_for_io_read_method(self, method_name: str):
         assert type_name
 
         asset_class_names: set[str] = {
-            t.__name__.lower().split("asset")[0]
+            camel_to_snake(t.__name__)[:-6]
```

Comment:
`non-blocking`
Can we revert this?

Reply:
Yeah, it doesn't look great in retrospect.

Reply:
✅